package.json

```diff
@@ -620,6 +619,7 @@
     "mocha": "^10.2.0",
     "mocha-junit-reporter": "^2.2.1",
     "mockttp": "^3.10.1",
+    "navigator.locks": "^0.8.6",
```

Comment:
Just moved from `dependencies` to `devDependencies`, since we still need it in our unit tests which run in Node (Node doesn't have `navigator.locks` support)